src/styles/blocks/section-title.scss

```diff
@@ -0,0 +1,18 @@
+.section-title {
+  margin: 0;
+  margin-bottom: 48px;
```

Comment:
don't add margins for bem block

Reply:
Hello, could you explain me? How can I use margin? Do I need to use another file, maybe the reset file, or something else?

Reply:
Or mixin?

Reply:
> don't add margins for bem block
Hello, could you explain me? How can I use margin? Do I need to use another file, maybe the reset file, or something else?
Or mixin?